src/types/code.ts

```diff
@@ -0,0 +1,8 @@
+import { type } from 'os';
+
+export type code = {
```

Comment:
```suggestion
export type Code = {
```

can we capitalize this type?